stdlib/src/builtin/string.mojo

```diff
@@ -1008,11 +1008,10 @@ struct String(
         Construct a String from several `Formattable` arguments:
 
         ```mojo
-        from testing import assert_equal
-
         var string = String.format_sequence(1, ", ", 2.0, ", ", "three")
-
-        assert_equal(string, "1, 2.0, three")
+        print(string) # "1, 2.0, three"
```

Comment:
Could you please remove these and the commented section below?

Reply:
do you mean removing the whole example or undoing my changes ?

Reply:
This particular `print` statement, as well as all the `#FIXME ` below that has large code examples. Since it's commented out, we cannot test it.

Reply:
> This particular print statement

if I do that then the user would only see
Examples:
```mojo
var string = String.format_sequence(1, ", ", 2.0, ", ", "three")
```
instead of seeing the result of the print, since the assert_equals gets hidden by the `%#`

> all the #FIXME  below that has large code examples

ok

Reply:
oh sorry, this is an example, nevermind 🤦

Reply:
So then you can either have an assert or the print. Is there a reason why this example needs to be updated at all?

Reply:
I found it ugly , IMO asserts shouldn't be used in examples and we should stick with print and showing with a comment what happens more similarly to what is done in Python. But I also like the idea of hiding an assert to ensure documentation is up to date with method behavior

Reply:
Okay, I guess I agree that print is probably nicer in an example. If you change it, please remove the commented out asserts below.

Reply:
They aren't commented, they will run as tests if the file [gets linked in `mojo test`](https://docs.modular.com/mojo/tools/testing#codeBlockContent_m3Ux), do you still want me to remove them?

Reply:
whoah, TIL, thanks, it's fine